book/src/dev/rfcs/0010-v5-transaction.md

```diff
@@ -0,0 +1,245 @@
+- Feature Name: `v5_transaction`
+- Start Date: 2021-03-11
+- Design PR: [ZcashFoundation/zebra#1886](https://github.com/ZcashFoundation/zebra/pull/1886)
+- Zebra Issue: [ZcashFoundation/zebra#1863](https://github.com/ZcashFoundation/zebra/issues/1863)
+
+# Summary
+[summary]: #summary
+
+Network Upgrade number 5 (`NU5`) introduces a new transaction type (transaction version 5). This document is a proposed design for implementing such a transaction version.
+
+# Motivation
+[motivation]: #motivation
+
+The Zebra software wants to be a protocol compatible Zcash implementation. One of the tasks to do this includes the support of the new version 5 transactions that will be implemented in the next network upgrade.
+
+# Definitions
+[definitions]: #definitions
+
+- `NU5` - the 5th Zcash network upgrade, counting from the `Overwinter` upgrade as upgrade zero.
+- `Orchard` - a new shielded pool introduced in `NU5`.
+- `Sapling` - a new shielded pool introduced in the 1st network upgrade. (`Sapling` is also the name of that network upgrade, but this RFC is focused on the `Sapling` shielded pool.)
+- `orchard data` - Data types needed to support orchard transactions.
+- `sapling data` - Data types needed to support sapling transactions.
+- `orchard transaction version` - Transactions that support orchard data. Currently only V5.
+- `sapling transaction version` - Transactions that support sapling data. Currently V4 and V5 but the data is implemented differently in them.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+V5 transactions are described by the protocol in the second table of [Transaction Encoding and Consensus](https://zips.z.cash/protocol/nu5.pdf#txnencodingandconsensus).
+
+We need the raw representation to Serialize/Deserialize the transaction however Zebra uses its own representation internally.
+
+V5 transactions are the only ones that will support orchard transactions with `Orchard` data types.
+
+V4 and V5 transactions both support sapling data but they are implemented differently. By this reason we need to split sapling data types into V4 and V5.
```

Comment:
```suggestion
V4 and V5 transactions both support sapling data but they are implemented differently. For this reason we need to split sapling data types into V4 and V5.
```

Reply:
Fixed in e7653681c6b63f3b26cfcbd2df7cfc26e573f5c8, because that paragraph also needed design updates.